jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -419,23 +418,18 @@ static String getServerName(Request request)
         if (uri.hasAuthority())
             return HostPort.normalizeHost(uri.getHost());
 
-        HostPort authority = request.getConnectionMetaData().getHttpConfiguration().getServerAuthority();
+        HostPort authority = request.getConnectionMetaData().getServerAuthority();
         if (authority != null)
             return HostPort.normalizeHost(authority.getHost());
```

Comment:
No need to normalize again, as `HostPort` normalizes in the constructor.

Reply:
@sbordet Fixed, but will need a re-review to get green tick back